.make-packages.js

```diff
@@ -40,7 +40,9 @@ fs.removeSync(PKG_ROOT);
 let rootPackageJson = Object.assign({}, pkg, {
   name: 'rxjs',
   main: './index.js',
-  typings: './index.d.ts'
+  typings: './index.d.ts',
+  module: './_esm5/index.js',
+  es2015: './_esm2015/index.js'
```

Comment:
It's hard to keep everything straight, but AFAICT this also makes the whole path-mappings.js thing unnecessary if you're using a newish version of webpack or rollup.

Would be cool to still include instructions (and maybe path-mappings still) for people who are stuck on older build pipelines.

Reply:
Yes, if users only import from `"rxjs"` and `"rxjs/operators"`, importing `"rxjs/index"` or (more realistically) `"rxjs/Rx"` would not be mapped.